crates/apollo-mcp-server/src/runtime/operation_source.rs

```diff
@@ -0,0 +1,104 @@
+use std::path::PathBuf;
+
+use schemars::JsonSchema;
+use serde::Deserialize;
+
+/// Source for loaded operations
+#[derive(Debug, Default, Deserialize, JsonSchema)]
+#[serde(tag = "type", rename_all = "snake_case")]
```

Comment:
"type" isn't really clear in the YAML file - maybe "source" would be better?
```suggestion
#[serde(tag = "source", rename_all = "snake_case")]
```